src/file-utils/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "edfi-lms-file-utils"
-version = "1.0.0-alpha.11"
+version = "1.0.0-beta.11"
```

Comment:
The name doesn't really matter... just fyi the only reason I had used `beta` while you were out was to avoid confusion with Luis's work. No problem leaving this.

Reply:
I went to beta because Poetry wasn't seeing my alpha.13 release on PyPi for some reason. It saw my previous bump to alpha.12 just fine, so don't know what the issue was but bumping to beta fixed it.  Probably makes more sense anyway to have it as the proper "latest".